src/Microsoft.TryDotNet.IntegrationTests/EditorTests.cs

```diff
@@ -356,6 +357,7 @@ public async Task when_user_code_in_editor_diagnostics_are_produced()
     }
 
     [IntegrationTestFact]
+    [Obsolete]

```

Comment:
Curious, why did we have to make these obsolete? Are these tests failing?

Reply:
If this is to prevent the warning that the tests are using an obsolete API, the preferable approach is to suppress that warning for the entire test project, since it's not really adding value here.